### PR TITLE
ENG-1818 adds deploy to new front door global stack

### PR DESCRIPTION
**Pull request by mstrisoline**
## Description

- deploys to both front door locations

## Testing required outside of automated testing?

- [x] Not Applicable

### Screenshots (if appropriate):

- [ ] Not Applicable

## Rollback / Rollforward Procedure

- [x] Roll Forward
- [ ] Roll Back

## Reviewer Checklist

- [ ] Description of Change
- [ ] Description of outside testing if applicable.
- [ ] Description of Roll Forward / Backward Procedure
- [ ] Documentation updated for Change

### CODE REVIEW

File: .github/workflows/pull-request.yml [hunk 1]
```diff
@@ -79,6 +81,7 @@ jobs:
         run: make setup-infra
         env:
           PULUMI_ACCESS_TOKEN: ${{ secrets.PULUMI_ACCESS_TOKEN }}
+          STACK_NAME: dev
```
Comment (djejaquino) [inline .github/workflows/pull-request.yml:84]: Why we need this?

**Review by djejaquino — APPROVED** [review]

**Review by djejaquino — CHANGES_REQUESTED** [review]
bundle is broken

File: .github/workflows/pull-request.yml (hunk 1 above)
Comment (mstrisoline) [reply .github/workflows/pull-request.yml]: Its not now, leftover from when i was doing a conditional on the domain.

Comment (mstrisoline) [reply .github/workflows/pull-request.yml]: I removed this.

**Review by djejaquino — APPROVED** [review]

**Pull request merged**